### DISCUSSION TITLE
Cant log in

### DISCUSSION THREAD

**Discussion by treetopteresa61** [mastodon/mastodon#22889] [General]
### Steps to reproduce the problem

1. Trying to log into Mastodon
2.Doesnt recognize me or my password.
3.Wont send email 
...

### Expected behaviour

Log into Mastodon

### Actual behaviour

Mastodon wont let me log in.

### Detailed description

I have a new computer and am trying to log into my acct, I wrote down my password so I know it is correct. 

### Specifications

Mastodon v.4.0.2

**Comment by rbairwell** [comment]
Hi, this Github repository is more for problems running the code/software rather than end-user support, but I'll try and help.

Each Mastodon server (also called an instance) is individually run and only mastodon.social and mastodon.online are actually operated by Mastodon gGmbh. Your Mastodon username will be in the format @[username@instance.tld](mailto:username@instance.tld) : if you go to https://instance.tld/ and try and sign in you should be able to request a new password- it may take a short time to come through depending on the volunteer run servers backlog.

Can't remember which server/instance you picked? Search your email for the "Welcome to Mastodon" email and it'll have your full Mastodon username/handle in the "Your full handle" section.

If you need further help, we'll need your full handle/username to be able to direct you to the correct instance or contact that instances server admin.

**Comment by midnightstarr** [comment]
Tried https://instance.tld by I get "This site can’t be reached Check if there is a typo in instance.tld.
If spelling is correct, [try running Windows Network Diagnostics](javascript:diagnoseErrors()).
DNS_PROBE_FINISHED_NXDOMAIN"

**Reply by mitexleo** [reply]
Which instance did you join? Is it mastodon.social ?

https://instance.tld is an example. TLD stands for Top Level Domain (.com, .net , .xyz etc.).

**Comment by NahNick** [comment]
> ### Steps to reproduce the problem
> 
> 1. Trying to log into Mastodon
> 
> 2.Doesnt recognize me or my password.
> 
> 3.Wont send email 
> 
> ...
> 
> 
> 
> ### Expected behaviour
> 
> Log into Mastodon
> 
> ### Actual behaviour
> 
> Mastodon wont let me log in.
> 
> ### Detailed description
> 
> I have a new computer and am trying to log into my acct, I wrote down my password so I know it is correct. 
> 
> ### Specifications
> 
> Mastodon v.4.0.2

**Reply by NahNick** [reply]
Ty
